src/scss/App.scss

```diff
@@ -0,0 +1,184 @@
+$white: #FFFFFF;
+
+@font-face {
+  src: url('../assets/fonts/Abadi-MT-Condensed-Extra-Bold.ttf');
+  font-family: Abadi;
+}
+
+@font-face {
+  src: url('../assets/fonts/AvenirNextLTPro-Cn.otf');
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url('../assets/fonts/nba-spurs.ttf');
+  font-family: spurs;
+}
+
+body {
+  margin: 0;
+  background-color: $background-color-main;
+}
+
+.app {
+  text-align: center;
+}
+
+.players {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.header-container {
+  display: flex;
+  height: 45px;
+  justify-content: space-evenly;
+  box-shadow: 1px -1px 2px $border-shadow;
+  font-family: Abadi;
+  list-style: none;
+  
+  .team {
+    position: relative;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background: $background-color-purple;
+
+    .logo-container {
+      display: flex;
+      border-radius: 24px;
+      box-shadow: 1px 1px 5px $border-dark-grey;
+      margin: 0 15px;
+      background-color: white;
+    }
+
+    button {
+      padding: 0 15px;
+      height: 100%;
+      border: none;
+      background: none;
+      color: $white;
+      font-family: spurs;
+      font-size: 22px;
+      letter-spacing: 1px;
+      text-transform: uppercase;
+      cursor: pointer;
+
+      i {
+        display: inline-block;
+        vertical-align: middle;
+        margin-left: 10px;
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 100%;
+      z-index: 5;
+      border-top: 45px solid $background-color-purple;
+      border-right: 20px solid transparent;
+    }
+  }
+
+  .links {
+    position: relative;
+    display: flex;
+    flex: 1.5;
+    justify-content: center;
+    align-items: center;
+    background-color: white;
+    font-family: Avenir;
+    font-size: 16px;
+    text-transform: uppercase;
+    
+    .link-container {
+      display: inline-block;
+      height: 100%;
+      padding: 0 25px;
+    }
+  
+    a {
+      display: inline-block;
+      height: calc(100% - 19px);    
+      padding-top: 15px;
+      color: black;
+      text-decoration: none;
+
+       &:hover {
+        height: calc(100% - 19px);
+        color: $grey-three;
+      }
+
+      &.active {
+        border-bottom: 4px solid $background-color-purple;
+        color: $background-color-purple;
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 100%;
+      border-top: 45px solid $white;
+      border-right: 20px solid transparent;
+    }
+  }
+
+  .user {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    padding-left: 30px;
+    color: $text-dark-grey;
+    background: $background-color-grey;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+  
+    > * div {
+      display: flex;
+      padding: 0 20px;
+      height: 100%;
+    }
+
+    button {
+      display: flex;
+      align-items: center;
+      padding: 15px;
```

Comment:
As you can see, the padding is still overflowing the div:

<img width="320" alt="Screen Shot 2020-02-13 at 2 20 55 PM" src="https://user-images.githubusercontent.com/623755/74470245-2c023c80-4e6c-11ea-928e-9abfa22f7f2d.png">

Let's change the padding like so:

```
padding: 0 15px;
height: 100%;
```

which will make it look like this ✅ :

<img width="326" alt="Screen Shot 2020-02-13 at 2 23 02 PM" src="https://user-images.githubusercontent.com/623755/74470384-6e2b7e00-4e6c-11ea-9d3e-918b2cde7a09.png">